apiserver/facades/client/modelconfig/modelconfig_test.go

```diff
@@ -5,22 +5,19 @@ package modelconfig_test
 
 import (
 	"github.com/juju/errors"
-	"github.com/juju/names/v4"
-	gitjujutesting "github.com/juju/testing"
-	jc "github.com/juju/testing/checkers"
-	gc "gopkg.in/check.v1"
-
 	"github.com/juju/juju/apiserver/facades/client/modelconfig"
 	apiservertesting "github.com/juju/juju/apiserver/testing"
-	"github.com/juju/juju/controller"
 	"github.com/juju/juju/core/constraints"
 	"github.com/juju/juju/environs/config"
-	"github.com/juju/juju/feature"
 	"github.com/juju/juju/provider/dummy"
 	_ "github.com/juju/juju/provider/dummy"
 	"github.com/juju/juju/rpc/params"
 	"github.com/juju/juju/state"
 	"github.com/juju/juju/testing"
+	"github.com/juju/names/v4"
+	gitjujutesting "github.com/juju/testing"
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
```

Comment:
This import order is not correct, please read the Import section in https://github.com/juju/juju/blob/develop/CONTRIBUTING.md

Reply:
The lint did not complain. Might have to do it manually. Thank you.
```
./main.sh static_analysis test_static_analysis_go

==> Checking for dependencies
==> Running subtest test_static_analysis_go for static_analysis suite
==> TEST BEGIN: static_analysis (tmp.9JE)
==> TEST SKIPPED: static copyright analysis
==> TEST SKIPPED: test doc go
===> [   ] Running: go
===> [ ✔ ] Success: go (19s)
===> [   ] Running: go tidy
===> [ ✔ ] Success: go tidy (0s)
==> TEST SKIPPED: static shell analysis
==> TEST SKIPPED: static python analysis
==> TEST SKIPPED: static schema analysis
==> TEST DONE: static_analysis (19s)
==> Cleaning up
====> Completed cleaning up jujus

==> Test result: success
==> Tests Removed: /home/capricorn/code/go/src/juju/tests/tmp.9JE
==> TEST COMPLETE
```

Reply:
Fixed manually

Reply:
Thx, now looks good